code/game/objects/items/rogueitems/keyrings.dm

```diff
@@ -152,7 +152,7 @@
 	keys = list(/obj/item/key/tavern/village, /obj/item/key/roomvi/village, /obj/item/key/roomv/village, /obj/item/key/roomiv/village, /obj/item/key/roomiii/village, /obj/item/key/roomii/village, /obj/item/key/roomi/village)
 
 /obj/item/storage/keyring/gatemaster
-	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/manor, /obj/item/key/walls)
+	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/manor, /obj/item/key/walls, /obj/item/key/keep_armory)
```

Comment:
Not sure Gatemaster needs armoury keys as above, though I suppose It would be better then MAA having armoury keys.